CustomCraftSML/ScannerMappings.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace CustomCraft2SML
+{
+    using System.Collections.Generic;
+
+    internal static class ScannerMappings
```

Comment:
Remove `ScannerMappings`. This will be handled in [SMLHelper v2.1](https://github.com/SMLHelper/SMLHelper/pull/75)